Contributor landing page

arun1595 opened the pull request:
<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->

##### Description
<!-- A description on what this PR aims to solve -->

**This is a WIP. Not ready for review yet.**

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] linter status: 100% pass
- [ ] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, ui, crypto, etc). -->
UI

##### Refers/Fixes
<!--
  Link to an issue if applicable. For example:
  If your PR fixes an issue  -> Fixes: #102
  If your PR refers an issue -> Refs: #101
-->

Fixes #1386

mbeacom commented:
This is live on staging:  https://stage.gitcoin.co/contributor

arun1595 commented on app/retail/templates/contributor_landing.html:
```diff
@@ -0,0 +1,101 @@
+            <h1>{% trans "What is Gitcoin?" %}</h1>
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center align-items-center whatisit mb-4 flex-wrap">
```
@thelostone-mc I thought of it but we've different content in the landing pages and I am unsure how to pass custom HTML to a template. Can you help me with that?

thelostone-mc replied:
It would look along the lines of [this](https://github.com/gitcoinco/web/blob/master/app/retail/views.py#L36-L62) 

We'd the it as an array of image + text and render them in the contributor_landing and the funder_landing page

Both would have it's own version of `whatisit` object (need a better name 😂 )

If it's too much of a trouble -> I'll take care of it later just leave a `TODO` for me

arun1595 replied:
@thelostone-mc Thanks for the explanation. I've refactored the code to include a 'What is Gitcoin' template. Do let me know how it looks.

thelostone-mc commented:
Lemme know when this good for review ^_^

arun1595 commented:
@thelostone-mc There are a few minor things missing but you can start your review when you get a chance. Thanks!

thelostone-mc commented:
The footer looks fixed to the bottom ! Derp derp

<img width="1440" alt="screen shot 2018-06-25 at 7 25 27 am" src="https://user-images.githubusercontent.com/5358146/41856038-f2c19722-7848-11e8-9df5-9ec796cf9188.png">

---

We need the add the vertical line.

<img width="1440" alt="screen shot 2018-06-25 at 7 34 02 am" src="https://user-images.githubusercontent.com/5358146/41856537-2f9f2604-784a-11e8-85ef-df2f5bbce669.png">

If you need you can reuse code from here https://gitcoin.co/how/contributor


---

Feels like we've got too many icons on one line. 
Should we limit it 6 per line ? @PixelantDesign 

<img width="1303" alt="screen shot 2018-06-25 at 7 32 47 am" src="https://user-images.githubusercontent.com/5358146/41856442-fac43bfe-7849-11e8-8406-bd7a23f9e8a1.png">


Could you resolve the conflicts too ? :P

thelostone-mc commented:
@PixelantDesign How it looks

![screencapture-0-0-0-0-8000-contributor-2018-06-25-07_36_19](https://user-images.githubusercontent.com/5358146/41856662-84d25966-784a-11e8-9569-af6449e2992c.png)

mbeacom commented:
@arun1595 Would you mind resolving the conflicts here?

thelostone-mc commented:
@arun1595 any chance we could get this in this week ?

arun1595 commented:
@thelostone-mc I've resolved the conflicts. Will address your comments shortly.

arun1595 commented:
@thelostone-mc I've addressed your comments, take a look when you get a chance.

@mbeacom Can you deploy this to staging so @PixelantDesign can review?

Thanks!

SaptakS commented:
The lastest is on staging right now: https://stage.gitcoin.co/
